setup.php

```diff
@@ -20,67 +20,114 @@
-								</div>
-								<div class="form-group col-xs-offset-1 col-xs-10">
-									<label for="password2">Repeat Password:</label>
-									<input type="password2" class="form-control" id="password2" name="password2"></input>
+					<div class="row">
+						<div class="panel panel-default col-xs-offset-1 col-xs-10">
+							<div class="panel-heading">
+								<h3 class="panel-title">Administrator Credentials</h3>
+							</div>
+							<div class="panel-body">
+								<p><?php echo $ProjectName ?> currently has an Administrator
+								which can configure the parameters of the software. Please
+								enter the user credentials below.</p>
+								<p>Make sure the credentials you use are secure and cannot
+								be guessed by employees and/or customers who may obtain
+								access to the service.</p>
+								<div class="row">
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="username">Username:</label>
+										<input type="text" class="form-control" id="username" name="username"></input>
+									</div>
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="password">Password:</label>
+										<input type="password" class="form-control" id="password" name="password"></input>
+									</div>
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="password2">Repeat Password:</label>
+										<input type="password" class="form-control" id="password2" name="password2"></input>
+									</div>
 								</div>
-
+									<button type="submit" class="btn btn-info col-xs-offset-1 center-block">Submit</button>
 							</div>
 						</div>
 					</div>
+				</form>
+			<?php }else{
+				class MyDB extends SQLite3{
+					function __construct(){
+					    $this->open($GLOBALS['DBname']);
+					}
+				}
+				$db = new MyDB();
+				if(!$db){
+					echo $db->lastErrorMsg();
+				} else {
+					$sql ="
+      				CREATE TABLE USERS
+				      (ID INTEGER PRIMARY KEY  AUTOINCREMENT   NOT NULL,
+				      username           TEXT    NOT NULL,
+				      password            TEXT     NOT NULL,
+				      priv         INT NOT NULL);";
+				}
+				$ret = $db->exec($sql);
+				if(!$ret){
+				    echo $db->lastErrorMsg();
+				} else {
+				    $sql = "
+				    CREATE TABLE PRODUCTS
+				    (ID INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+				    name TEXT NOT NULL,
+				    price REAL NOT NULL)";
+				    $ret = $db->exec($sql);
+				   	if(!$ret){
+				      	echo $db->lastErrorMsg();
+				   	}
+				   	$sql = "INSERT INTO users(username,password,priv) VALUES ('".$_POST['username']."','".$_POST['password']."',5);";
+				   	$ret = $db->exec($sql);
+				   	if(!$ret){
+				      	echo $db->lastErrorMsg();
+				   	}
+				   	$db->close();	   
+				}
+			}
+		?>
+			<div class="row">
+				<div class="panel panel-default">
+					<div class="panel-body text-justify">
+						<p class="text-center">Database created successfully</p>
```

Comment:
If the database creation fails, you are `echo`ing the error message. However, this message is still visible from what I can tell, so the user will see some error messages and then this. 

Also, you may want to visualize this differently: a success can use the Bootstrap `success` class/tag to make the result more clear, while an error should be printed beautifully with the `danger` tag included.